src/app/app-routing.module.ts

```diff
@@ -1,7 +1,13 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { FormComponent } from './pages/form/form.component';
+import { ListComponent } from './pages/list/list.component';
 
-const routes: Routes = [];
+const routes: Routes = [
+  { path: 'form', component: FormComponent },
```

Comment:
dodatkowo powinien być dodany `initial path`
U ciebie by to było 
` { path: '', redirectTo: '/form', pathMatch: 'full' },`
Dzięki temu zawsze na starcie będziesz mieć widok z FormComponent

Reply:
https://stackoverflow.com/questions/37605119/angular2-router-angular-router-how-to-set-default-route

Reply:
Tak jak w odpowiedzi ze stacka powyżej warto też dodać `route` dla `invalid routes`
u ciebie to będzie 
`{ path: '**', redirectTo: '/form', pathMatch: 'full' },`